core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java

```diff
@@ -503,7 +503,7 @@ private void crossSegmentShadowLine(
      * @mavengroupid $GroupId$
      * @mavenartifactid $ArtifactId$
      */
-    private final  class CoordinatesParam {
```

Comment:
To shorthen the length of parameter  count passed to the function  and
reduce the  line length. I wanted to bundle the coordinates in a class an
pass them via the object. .

On Fri, Jun 24, 2016 at 12:42 PM, Stéphane Galland <notifications@github.com
> wrote:

> In
> core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java
> <https://github.com/gallandarakhneorg/afc/pull/156#discussion_r68376060>:
>
> > @@ -503,7 +503,7 @@ private void crossSegmentShadowLine(
> >       * @mavengroupid $GroupId$
> >       * @mavenartifactid $ArtifactId$
> >       */
> > -    private final  class CoordinatesParam {
>
> Why have you introduced this class? Why not using standard function
> argument passing?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/156/files/2adeea42b5a5106662baad9d732535e7bf0227c9..2d4d424579ba7f3d74375e70266100a27de3a354#r68376060>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BJpQzl6cQESfoaUbOLr1orZfcULeks5qO6ZqgaJpZM4I9Db2>
> .
>